homeassistant/components/google_assistant/report_state.py

```diff
@@ -80,7 +80,16 @@ async def async_entity_state_listener(
         ):
             return
 
-        if (notifications := entity.notifications_serialize()) is not None:
+        # For doorbell events we use notitication and we need to synchronize
+        # them using a `SYNC` response together with other state changes.
+        # We only want to trigger on this state attribute (timestamp) to avoid false positives.
```

Comment:
"We only trigger on changes in the state value, not attributes. This is mainly designed for our event entity types"